torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -46,6 +46,7 @@ struct Welford {
   T mean = T(0);
   T m2 = T(0);
   T weight = T(0);
+  int64_t index = 0;
```

Comment:
Can we reuse the `weight`? Looks like they have same value but different is the type(`int64` or `Vectorized<T>`).

Reply:
Can not get index value from weight directly, e.g. weight[0], with T = at::vec::Vectorized<...> as `operator[] ... = delete`.
`int64_t index = 0`  has almost no impact on performance in scalar tests.

Reply:
Now that I think about it, it might be true that in the CPU backend the `weight` vector always contains that same value in every spot, which should mean you could ignore the weight entirely until you get to the stage of combining accumulators.

Reply:
Yes, `weight` vector always contains that same value as index.